app/styles/components/_nav.scss

```diff
@@ -102,6 +99,20 @@ $nav-break-mobile: 798px;
   transition: all 0.6s;
 }
 
+.Nav.Nav--onTopIndex {
```

Comment:
I'm not sure this is okay. The rules for this class are mostly defined above because they are shared with the Nav--onTop except this one.